core/tee/tee_ree_fs.c

```diff
@@ -476,7 +476,7 @@ static TEE_Result get_dirh(struct tee_fs_dirfile_dirh **dirh)
 		}
 	}
 	assert(ree_fs_dirh);
-	ree_fs_dirh_refcount++;
+	assert(ree_fs_dirh_refcount);
```

Comment:
Is there any sense in this `assert` if you increase refcount in the beginning of the function?